crates/control/src/motion/look_at.rs

```diff
@@ -90,55 +85,49 @@ impl LookAt {
             self.last_glance_direction_toggle = Some(cycle_start_time);
         }
 
-        let (target, camera) = match head_motion {
-            HeadMotion::LookAt { target, camera } => (target, camera),
+        let (target, pixel_target, camera) = match head_motion {
```

Comment:
you can deref here: `match *head_motion`

Reply:
Done